apex-bin/pom.xml

```diff
@@ -0,0 +1,243 @@
+              </execution>
+              <execution>
+                <phase>package</phase>
+                <id>wget-kryo</id>
+                <goals>
+                  <goal>wget</goal>
+                </goals>
+                <configuration>
+                  <url>https://raw.githubusercontent.com/EsotericSoftware/kryo/kryo-2.24.0/license.txt</url>
+                  <outputDirectory>${project.build.directory}/licenses</outputDirectory>
+                  <outputFileName>kryo-LICENSE.txt</outputFileName>
+                </configuration>
+              </execution>
+              <execution>
+                <id>wget-mbassador</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>wget</goal>
+                </goals>
+                <configuration>
+                  <url>https://raw.githubusercontent.com/bennidi/mbassador/mbassador-1.1.9/LICENSE</url>
+                  <outputDirectory>${project.build.directory}/licenses</outputDirectory>
+                  <outputFileName>mbassador-LICENSE.txt</outputFileName>
+                </configuration>
+              </execution>
+              <execution>
+                <id>wget-minlog</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>wget</goal>
+                </goals>
+                <configuration>
+                  <url>https://raw.githubusercontent.com/EsotericSoftware/minlog/1.2/license.txt</url>
+                  <outputDirectory>${project.build.directory}/licenses</outputDirectory>
+                  <outputFileName>minlog-LICENSE.txt</outputFileName>
+                </configuration>
+              </execution>
+              <execution>
+                <id>wget-jersey</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>wget</goal>
+                </goals>
+                <configuration>
+                  <url>https://raw.githubusercontent.com/jersey/jersey/master/LICENSE.txt</url>
+                  <outputDirectory>${project.build.directory}/licenses</outputDirectory>
+                  <outputFileName>jersey-LICENSE.txt</outputFileName>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <version>3.0.0</version>
+            <configuration>
+              <descriptors>
+                <descriptor>src/assemble/bin.xml</descriptor>
+              </descriptors>
+            </configuration>
+            <executions>
+              <execution>
+                <id>make-assembly</id>
+                <phase>package</phase>
+                <goals>
+                  <goal>single</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>apache-release</id>
```

Comment:
This isn't required here. To the contrary, the dist build should probably only run under this profile (parent pom.xml).

Reply:
Make sense... Moved the "dist" module under modules tag of apache-release profile.